Makefile

```diff
@@ -1,5 +1,5 @@
 OPTFLAGS = -march=native -mtune=native -O2
-CXXFLAGS += -g -Wall -Wextra -Werror -Wfatal-errors -Wno-unused-parameter -std=c++11 -fPIC -Wno-unused-variable
+CXXFLAGS += -g -Wall -Wextra -Wno-unused-parameter -std=c++11 -fPIC -Wno-unused-variable
```

Comment:
[dnb - does not block] In the long run I'm against removing ``-Werror``, but in the short term, let's leave this removed, and then have a followup task to disallow any compiler warnings. Thoughts?